src/forms/signpost/SignPost.tsx

```diff
@@ -156,16 +163,20 @@ export class SignPost extends React.PureComponent<SignPostProps> {
 
     buildSignPostTriggerButton = () => {
         const {isOpen} = this.state;
-        const {openAt} = this.props;
-        return (
+        const {openAt, customSignPost} = this.props;
+
+        return customSignPost ? (
+            <div
+                onClick={this.handleOnClick}
+                className={classNames([signPostAction, signPostTrigger, isOpen && signPostActive])}
+            >
+                {openAt}
+            </div>
+        ) : (
             <Button
-                className={classNames([
-                    "signpost-action", //prettier
-                    "btn-small",
-                    "btn-link",
-                    "signpost-trigger",
-                    isOpen && "active",
-                ])}
+                className={classNames([signPostAction, signPostTrigger, isOpen && signPostActive])}
```

Comment:
you can create single constant for classNames and use it here and above line like,
const signPostClasses = classNames([signPostAction, signPostTrigger, isOpen && signPostActive, <Prop value for user defined className>])